packages/components/e2e/component-objects/src/main/java/org/talend/component/Component.java

```diff
@@ -41,6 +43,21 @@ public Component(WebDriver driver, String name, String selector) {
     }
 
     /**
+     * Component constructor
+     *
+     * @param driver Selenium WebDriver
+     * @param root Component root element
+     */
+    public Component(WebDriver driver, String name, WebElement root) {
+        LOGGER.info("Component " + name + " from provided root");
+        this.driver = driver;
+        this.name = name;
+        this.root = root;
+
+        this.driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```

Comment:
This was set by the TFD QA, I just kept it.
This is their settings. But now that you point it, it should not be there, it overrides the app test config.
I'll remove it.